go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/tektoncd/catalog
+
+go 1.16
```

Comment:
@chmouel
This looks like what's causing the failure

Shall I move this down to `go 1.13` like plumbing or shall I use `go 1.15` as that seems to be what's available:

```
>> go version
go version go1.15.2 linux/amd64
```

https://storage.googleapis.com/tekton-prow/pr-logs/pull/tektoncd_catalog/663/pull-tekton-catalog-build-tests/1371435036477231105/build-log.txt

Or should I look at bumping the go version used to `go 1.16`?

Reply:
let's keep the same as what we have in plumbing?

Reply:
sure

Reply:
moved to `1.13`